.schema/config.schema.json

```diff
@@ -843,6 +843,12 @@
               "$ref": "#/definitions/retry"
             }
           }
+        },
+        "cache_ttl": {
```

Comment:
I'd like to propose another structure:

```
cache:
  ttl:
```

This would allow us to add more keys later on:

```
cache:
  strategy: ristretto # groupcache? lru?
  size: ..
  ...
```